src/Sulu/Component/Content/Types/Rlp/Mapper/PhpcrMapper.php

```diff
@@ -449,7 +455,6 @@ function ($resourceLocator, NodeInterface $historyNode) use (
                 ) {
                     // delete history nodes
                     $this->deleteByNode($historyNode, $session, $webspaceKey, $languageCode, $segmentKey);
-                    $session->save();
```

Comment:
Any suggestions where this might have been necessary? Everything I tested was working so far. /cc @wachterjohannes @chirimoya